evalml/objectives/standard_metrics.py

```diff
@@ -369,6 +390,7 @@ class LogLossBinary(BinaryClassificationObjective):
     score_needs_proba = True
     perfect_score = 0.0
     is_bounded_like_percentage = False  # Range [0, Inf)
+    expected_range = [0, 1]
```

Comment:
Although the range of log loss is (0, inf), we choose to set its expected range much smaller. I chose [0, 1] since we ideally expect the models to output very similar scores throughout the different CV folds. If the scores are too different (in this case, difference >= 0.5), we want to warn users that there could be high variance